src/pages/nomad/downloads/style.module.css

```diff
@@ -3,8 +3,7 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-.card {
-	background-color: var(--token-color-surface-faint);
-	border-color: var(token-color-border-strong);
-	color: var(--token-color-foreground-primary);
+.downloadLink {
+	margin-left: -11px;
+	margin-top: -6px;
```

Comment:
This is to account for the button padding which if unaccounted for misaligns the button with the rest of the inline alert content. I'm not sure how we've worked around this in the past. 

<img width="314" alt="Screenshot 2023-06-28 at 2 32 43 PM" src="https://github.com/hashicorp/dev-portal/assets/36613477/d7838cc2-4aee-49c0-a899-b165021a994b">